integration-cli/docker_cli_events_test.go

```diff
@@ -18,13 +18,12 @@ import (
 )
 
 func (s *DockerSuite) TestEventsTimestampFormats(c *check.C) {
-	image := "busybox"
+	name := "events-time-format-test"
 
 	// Start stopwatch, generate an event
-	time.Sleep(1 * time.Second) // so that we don't grab events from previous test occurred in the same second
 	start := daemonTime(c)
-	dockerCmd(c, "tag", image, "timestamptest:1")
-	dockerCmd(c, "rmi", "timestamptest:1")
+	time.Sleep(1 * time.Second) // so that first event occur in different second from since (just for the case)
```

Comment:
Maybe I'm too paranoid, but I think it's better to make these sleeps slightly longer than 1 second. The daemon could be on a separate machine (or scheduled on a different core) from the test runner, and if there is a little bit of clock skew, it's theoretically possible for this sleep to last a full second, but for the daemon log not to roll over to the next second.